### ISSUE TITLE
Nx plugin doesn't look in `package.json`

### ISSUE THREAD

**Issue by alecmev** [webpro-nl/knip#598]
Since Nx is more or less a replacement for good ol' scripts, I keep its project-specific configuration in `package.json`:

```jsonc
{
  "name": "foo",
  "nx": {
    "targets": {
      "fun": {
        "executor": "nx:run-commands",
        "options": {
          "command": "rimraf /"
        } 
      }
    }
  }
}
```

Currently the plugin doesn't check it for configuration:

https://github.com/webpro-nl/knip/blob/1ecdf800329dd9850771cbf270f451f85cff50b8/packages/knip/src/plugins/nx/index.ts#L13

So Knip errors with:

```
Unused devDependencies
                   
rimraf              package.json
```

Just in case, it has the exact same shape as `project.json`, just one level deeper, under `nx`.

**Comment by webpro** [comment]
Fixed in #598.

**Comment by alecmev** [comment]
Nit: Fixed in #880 😛
